docs/modules/user-guide/partials/proc_adding-projects-to-a-devfile.adoc

```diff
@@ -2,8 +2,103 @@
 = Adding projects to a devfile
 
 [role="_abstract"]
-This section describes how to add one or more projects to a devfile.
-
+This section describes how to add one or more projects to a devfile. Each starter project can contain three different objects:
+
+* git
+* github
+* zip
+
+See the following tables for project properties in a devfile:
+
+.starterProject object
+[cols="1,1,1,1"]
+|===
+|Key |Type| Required| Description
+
+|name
+|string
+|no
+|The name of your devfile.
+
+|description
+|string
+|no
+|The description of your starterProject.
+
+|clonePath
+|string
+|no
+|The path relative to the root of your projects. Clone your projects into this path.
+|===
+
+
+.gitObject
```

Comment:
git object fields in this table are old. They do not match the schema. Please refer to the schema in https://docs.devfile.io/devfile/2.1.0/user-guide/api-reference.html to reflect it.

Reply:
Please confirm I made the changes correctly, thanks!

Reply:
I didn't notice this before but should the title be `.git object` instead of `.gitObject` to match the `starterProjects` section title?

Reply:
resolving as I have made the change.